packages/Search/examples/bvh_driver/bvh_driver.cpp

```diff
@@ -139,12 +150,8 @@ int main_( Teuchos::CommandLineProcessor &clp, int argc, char *argv[] )
         Kokkos::View<DataTransferKit::Within *, DeviceType> queries(
             Kokkos::ViewAllocateWithoutInitializing( "queries" ), n_queries );
         // radius chosen in order to control the number of results per query
-        // NOTE: minus "1+sqrt(3)/2 \approx 1.37" matches the size of the boxes
-        // inserted into the tree (mid-point between half-edge and
-        // half-diagonal)
         double const r = 2. * std::cbrt( static_cast<double>( n_neighbors ) *
-                                         3. / ( 4. * M_PI ) ) -
-                         ( 1. + std::sqrt( 3. ) ) / 2.;
+                                         3. / ( 4. * M_PI ) );
```

Comment:
I'm not sure why you removed this. It seems to me that this instead should be hardened to deal with more cases.